### ISSUE TITLE
Not able to open safari developer mode for investigating issue on ios 16.4

### ISSUE THREAD

**Issue by FrostFamil** [FormidableLabs/react-native-app-auth#844]
## Issue

Not able to open safari developer mode for investigating issue on ios 16.4.
I assume new Webkit changes introduced with ios 16.4 and now its not able to do Inspection of Web Content in Apps using safari developer mode.
---

## Environment

* **Your Identity Provider**: `e.g. IdentityServer 4 / Okta / Azure`
* **Platform that you're experiencing the issue on**: `iOS`
* **Are you using Expo?** No

**Comment by Yannick-LMI** [comment]
We are reproducing this problem when running automation tests on IOS 16.4.x, the Safari View controller is not reachable anymore.

**Comment by FrostFamil** [comment]
Did you find any work around for this?

**Comment by Yannick-LMI** [comment]
> Did you find any work around for this?

@FrostFamil no, there is one for the WebView (inspectable flag new in IOS 16.4), but nothing for SafariViewController.

**Comment by FrostFamil** [comment]
Yeah its problematic for me too, can find a way to investigate a bug which is directly related to this SafariViewController  @Yannick-LMI

**Comment by apexskier** [comment]
More context - https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/
